File: backend-springboot/src/main/java/com/vaultx/banking/dto/CustomerLoginDto.java
package com.vaultx.banking.dto;

import jakarta.validation.constraints.NotBlank;

public class CustomerLoginDto {
    @NotBlank(message = "Email or phone is required")
    private String emailOrPhone;

    @NotBlank(message = "Password is required")
    private String password;

    public String getEmailOrPhone() { return emailOrPhone; }
    public void setEmailOrPhone(String emailOrPhone) { this.emailOrPhone = emailOrPhone; }

    public String getPassword() { return password; }
    public void setPassword(String password) { this.password = password; }
}

File: backend-dotnet/VaultX.Banking.API/DTOs/CustomerLoginDto.cs
using System.ComponentModel.DataAnnotations;

namespace VaultX.Banking.API.DTOs;

public class CustomerLoginDto
{
    [Required(ErrorMessage = "Email or phone is required")]
    public string EmailOrPhone { get; set; } = string.Empty;

    [Required(ErrorMessage = "Password is required")]
    public string Password { get; set; } = string.Empty;
}
